web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -613,43 +612,53 @@ const TeamReviews = ({ onBack, periodId }) => {
         },
         data: {
           ...period,
-          reviewStatus: ReviewStatus.OPEN
+          reviewStatus
         }
       });
       if (res.error) throw new Error(res.error.message);
       onBack();
     } catch (err) {
-      console.error('TeamReviews.jsx deleteReviewer:', err);
+      console.error('TeamReviews.jsx updateReviewPeriodStatus:', err);
     }
   };
 
+  // This is only used for debugging.
+  const logAssignments = assignments => {
+    if (currentMembers.length === 0) return;
+    for (const assignment of assignments) {
+      const reviewee = currentMembers.find(m => m.id === assignment.revieweeId);
+      const reviewer = currentMembers.find(m => m.id === assignment.reviewerId);
+      if (reviewee && reviewer) {
+        console.log(reviewee.name, 'is reviewed by', reviewer.name);
+      }
+    }
+  }
+
   const requestApproval = async () => {
     const msg = validateReviewPeriod(period);
     setValidationMessage(msg);
     if (msg) return;
 
-    const visibleIds = new Set(visibleTeamMembers().map(m => m.id));
-    const unapproved = assignments.filter(
-      a => !a.approved && visibleIds.has(a.revieweeId)
-    );
-    /* For debugging ...
-    for (const assignment of unapproved) {
-      const reviewee = currentMembers.find(m => m.id === assignment.revieweeId);
-      const reviewer = currentMembers.find(m => m.id === assignment.reviewerId);
-      console.log(reviewer.name, 'reviewing', reviewee.name, 'is unapproved.');
-    }
-    */
-    setUnapproved(unapproved);
-    setConfirmationText(
-      unapproved.length === 0
-        ? 'Are you sure you want to launch the review period?'
-        : unapproved.length === 1
-          ? 'There is one visible, unapproved review assignment. ' +
+    if (period.reviewStatus === ReviewStatus.PLANNING) {
```

Comment:
When the review period is in the PLANNING state, we just want to bump the state to AWAITING_APPROVAL.